### PR TITLE
✨ The client app can receiving device twin message

### PR DESCRIPTION
**Pull request by yuwzho**

### CODE REVIEW

File: main.c [hunk 1]
```diff
@@ -147,7 +182,7 @@ IOTHUBMESSAGE_DISPOSITION_RESULT receiveMessageCallback(IOTHUB_MESSAGE_HANDLE me
     strncpy(temp, buffer, size);
     temp[size] = '\0';
 
-    printf("Receiving message: %s\r\n", temp);
+    (void)printf("Receiving message: %s\r\n", temp);
```
Comment (ZhijunZhao) [inline main.c]: LogInfo?

File: main.c [hunk 2]
```diff
@@ -23,6 +23,8 @@ const char *notFound = "\"No method found\"";
 static bool messagePending = false;
 static bool sendingMessage = true;
 
+static int interval = INTERVAL;
```
Comment (ZhijunZhao) [inline main.c]: I think you can use INTERVAL directly.

Comment (yuwzho) [inline main.c]: this interval can be changed by the device twin method

File: main.c (hunk 1 above)
Comment (yuwzho) [inline main.c]: I think the LogInfo can be closed by some flag, but in the C2D message, it should always print out.

**Pull request merged**